### ISSUE TITLE
[Suggestion] Module paste creator

### ISSUE THREAD

**Issue by lc4ro** [MeteorDevelopment/meteor-client#2967]
### Describe the feature

instead of the favorite paste allow the user to create a paste of modules

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by Lovecrafts-Cat** [comment]
Can you rephrase that in a way that actually makes sense?

**Comment by I-Got-You-Dead** [comment]
> Can you rephrase that in a way that actually makes sense?

I understand it, it’s just something meteor already does

**Comment by I-Got-You-Dead** [comment]
This kid is kinda retarded ngl

**Comment by Lovecrafts-Cat** [comment]
> I understand it, it’s just something meteor already does

Does he just want a way to copy/paste configs?
That's a thing already.

**Comment by I-Got-You-Dead** [comment]
> > I understand it, it’s just something meteor already does
> 
> Does he just want a way to copy/paste configs? That's a thing already.

Yea pretty much
